packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -164,9 +164,17 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
   const mergedTextContainerStyle = mergeStyles(textContainerStyle, styles?.textFieldContainer);
   const mergedTextFieldStyle = concatStyleSets(textFieldStyle, {
     fieldGroup: styles?.textField,
-    errorMessage: styles?.systemMessage
+    errorMessage: styles?.systemMessage,
+    suffix: {
+      backgroundColor: 'transparent',
+      // Remove empty space in the suffix area when adding newline-style buttons
+      display: props.inlineChildren ? 'flex' : 'contents',
+      padding: '0px 0.25rem'
```

Comment:
I would suggest either changing the `0px` to be `0rem` or just `0` since we don't want pixels in our styles.